packages/cordova/config.xml

```diff
@@ -84,8 +82,8 @@
                 <string>UIInterfaceOrientationLandscapeRight</string>
             </array>
         </config-file>
+        <splash src="res/ios/splash/Default@3x~universal~anyany.png"/>
     </platform>
-    <plugin name="cordova-plugin-app-version" spec="~0.1.9" />
-    <plugin name="cordova-plugin-backbutton" spec="~0.3.0" />
-    <plugin name="ionic-plugin-keyboard" spec="~2.2.1" />
-</widget>
+    <plugin name="cordova-plugin-app-version" spec="~0.1.9"/>
+    <plugin name="cordova-plugin-backbutton" spec="~0.3.0"/>
```

Comment:
Shouldn't `cordova-plugin-keyboard` be here?

Reply:
Good question. I'm not sure if it's need but we can add it just to be sure. Would you take care of that please?

Reply:
Since there are many other plugins that don't show here and I assume this already worked, I'd rather not add a change without being able to test it through again.